challenges/programmer-humour/script.js

```
@@ -0,0 +1,14 @@
+    fetch('https://xkcd.vercel.app/?comic=latest')
+        .then(function(response) {
+            return response.json();
+        })
+    .then(function(data){
+        let image = document.createElement('img');
+        const rootEl = document.getElementById("root");
+      image.setAttribute("src", data.img);
+      rootEl.appendChild(image);
+      console.log(data)
+  })
+  .catch(err => alert('Something went wrong'));
```

Same suggestion as before with the `catch` method swallowing the `err` variable and hiding the actual error, make the error visibile somewhere!